Makefile

```diff
@@ -17,12 +17,13 @@ build: test
 	@cd /tmp/phpmetrics-build && php build.php
 	@cp /tmp/phpmetrics-build/build/phpmetrics.phar build/phpmetrics.phar
 	@rm -Rf /tmp/phpmetrics-build
-	
+
 	@echo Testing phar
 	./vendor/bin/phpunit -c phpunit.xml.dist --group=binary &&	echo "Done"
 
 	@echo Releasing sources
 	@sed -i -r "s/(v[0-9]+\.[0-9]+\.[0-9]+)/`semver tag`/g" bin/phpmetrics
```

Comment:
If I'm getting this properly, shouldn't this "seds" be before the generation of the .phar? 

My generated phar is getting the version 1.8.2 instead of the 1.8.3